src/ElectricPokemon.java

```diff
@@ -0,0 +1,188 @@
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon{
+    ElectricPokemon(String name, float weight, float height, String food, String sound){
+        super(name,  weight,  height,  food,  sound);
+    }
+
+    @Override
+    public void setName(String name) {
```

Comment:
This override method is unnecessary. If you leave it out, the function would be the same. You only cal the `super.setName()`, so why not let the call be redirected to the super directly.